addons/xpaas/v3.10/xpaas-streams/eap64-image-stream.json

```diff
@@ -0,0 +1,194 @@
+{
```

Comment:
@agajdosi We are explicitly told long back to remove eap6.x from the template and from the image stream, if you see 3.9 one those are not part of addon.

Reply:
@praveenkumar Thanks mate, completely missed that. Will remove that.